modules/microshift-adding-app-rpms-to-blueprint.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-adding-app-rpms-to-blueprint_{context}"]
+= Adding application RPMs to a blueprint
+
+.Prerequisites
+* You have root access to the host.
+* Workload or application RPMs exist in the `~/rpmbuild/RPMS` directory.
+
+To add application RPMs to a blueprint, you must create a local repository that Image Builder can use to create the ISO. With this procedure, the required container images for your workload can be pulled over the network.
+
+. Create a local RPM repository by running the following command:
++
+[source,terminal]
+----
+$ sudo createrepo ~/rpmbuild/RPMS/
+----
+
+. Give Image Builder access to the RPM repository by running the following command:
++
+[source,terminal]
+----
+$ sudo chmod a+rx ~
+----
+
+[NOTE]
+----
+You must ensure that Image Builder has all of the necessary permissions to access all of the files needed for image building, or the build cannot proceed.
+----
+
+. Create the blueprint file, `repo-local-rpmbuild.toml`, with this example content:
```

Comment:
In the preview render, the numbering start here again with 1, but it should be 3? Maybe it has something to do with the previous comment?